solver/pb/ops.proto

```diff
@@ -309,6 +309,8 @@ message FileAction {
 		FileActionMkDir mkdir = 6;
 		// FileActionRm removes a file
 		FileActionRm rm = 7;
+		// FileActionSymlink creates a symlink
+		FileActionSymlink symlink = 8;
```

Comment:
We should also add a capability https://github.com/moby/buildkit/blob/master/solver/pb/caps.go.

Reply:
Can you be more specific? What exactly needs to be added and why?

Reply:
This is done

I checked with @cpuguy83 but you may want to double-check the changes to make sure everything is there